avaframe/out3SimpPlot/outAIMEC.py

```diff
@@ -0,0 +1,547 @@
+        if not os.path.exists(os.path.dirname(outname_fin)):
+            os.makedirs(os.path.dirname(outname_fin))
+        fig.savefig(outname_fin, transparent=True)
+
+    if cfgFlags.getboolean('plotFigure'):
+        plt.show()
+    else:
+        plt.ioff()
+
+    plt.close(fig)
+
+
+def result_write(cfgPath, cfgSetup, resAnalysis):
+    """
+    This function is used as standart output file
+
+    example path: 'log/doublestep_5_5m_calibration_results.txt'
+    example header: 'runout, maxvel, rho, mu, tau_0, R_s^0, kappa, R, B, q \n'
+
+    INPUT: data, path, header
+    """
+
+    project_name = cfgPath['project_name']
+    path_name = cfgPath['path_name']
+    dem_name = os.path.basename(cfgPath['demSource'])
+    data_name = [os.path.basename(name) for name in cfgPath['pressurefileList']]
+    domainWidth = cfgSetup['domainWidth']
+    pressureLimit = cfgSetup['pressureLimit']
+
+    runout = resAnalysis['runout']
+    AMPP = resAnalysis['AMPP']
+    MMPP = resAnalysis['MMPP']
+    AMD = resAnalysis['AMD']
+    MMD = resAnalysis['MMD']
+    deltaH = resAnalysis['deltaH']
+    elevRel = resAnalysis['elevRel']
+    relMass = resAnalysis['relMass']
+    entMass = resAnalysis['entMass']
+    growthIndex = resAnalysis['growthIndex']
+    growthGrad = resAnalysis['growthGrad']
+
+    legend = ['fileNr', 'runout', 'elevRel', 'deltaH', 'AMPP',
+              'MMPP', 'entMass', 'growthIndex', 'AMD', 'MMD']
+    resfile = [runout, elevRel, deltaH, AMPP, MMPP, entMass, growthIndex, AMD, MMD]
+
+    header = ''.join(['project_name: ',  project_name, '\n',
+                      'path: ', path_name, '\n',
+                      'dhm: ', dem_name, '\n',
+                      'domain_width: ', str(domainWidth), '\n',
+                      'pressure_limit: ', str(pressureLimit), '\n',
+                      'release_mass: ', str(relMass[0]), '\n'])
+
+    outname = ''.join([cfgPath['pathResult'], os.path.sep,
+                       'Results_pl', str(pressureLimit),
+                       '_w', str(domainWidth), '.txt'])
+
+
+#    chekc if folder exists / create
+    if not os.path.exists(os.path.dirname(outname)):
+        os.makedirs(os.path.dirname(outname))
+
+    output = resfile
+    log.info('write output file: %s' % outname)
+    fid = open(outname, 'w')
+    fid.write(header)
+    # write table legend
+    for j in range(len(legend)):
+        fid.write('{:<12s}'.format(legend[j]))
+    fid.write('\n')
+    # write table values
+    for i in range(len(output[0])):
+        tmp = os.path.basename(data_name[i])
+        name = os.path.splitext(tmp)[0]
+        fid.write('{:<12s}'.format(name))
+        for j in range(len(output)):
+            try:
+                fid.write('{:<12.3f}'.format(output[j][i]))
+            except:
+                fid.write('{:<12}'.format('NaN'))
+        fid.write('\n')
+    fid.close()
+
+    log.info('File written: %s' % outname)
+
+
+def colorvar(k, k_end, colorflag, disp=0):
```

Comment:
Not important. but:
could this be done by a function provided by our plotting lib? (matplotlib?)

Reply:
I ll have a look at it